kubeflow/instance/kustomize/iap-ingress/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 patchesStrategicMerge:
 - iap-ingress-config.yaml
 - service-accounts.yaml
+- service.yaml
 resources:
 - ../../../upstream/manifests/gcp/iap-ingress/v3 # {"$kpt-set":"gcp-iap-ingress-v3"}
```

Comment:
Would you mind contributing to this upstream manifest instead of here?
Because instance folder should only include personal customizations

Reply:
@Bobgy I created this other PR https://github.com/kubeflow/manifests/pull/1579

I'm not familiar how the unit tests work. Let me know if I need to fix / adjust anything